docker/runtime/bash/http-trigger/startup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ -f /kubeless/requirements ]]; then
+ install_packages -y $(cat /kubeless/requirements)
+fi
+
+echo "Starting up runtime for bash functions"
+python /kubeless-shell.py 
```

Comment:
```sh
exec python /kubeless-shell.py
```
The `exec` is important so k8s' terminate signals go to the python process rather than this shell wrapper.